### PR TITLE
docs: Update template to ask for dependency versions

### PR DESCRIPTION
**Pull request by romanblanco**
Suggestion to ask contributors for more of their environment.

Also, adding a few minor changes in comments/texts. No change that would influence behavior.

## Problem
Could be easier to ask for package versions right away.

## Solution
Add field to note package versions into issue template.

(using the list of dependencies requested at https://github.com/rswag/rswag/issues/559#issuecomment-1261202414)

### This concerns this parts of the Open API Specification:
-

### The changes I made are compatible with:
- [x] OAS2
- [x] OAS3
- [x] OAS3.1

### Related Issues
-

### Checklist
- [ ] Added tests
- [x] Changelog updated
- [ ] Added documentation to README.md

### Steps to Test or Reproduce
-

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -4,8 +4,8 @@ A clear and concise description of what the problem is.
 ## Solution
 A clear and concise description of what the solution is.
 
-### This concerns this parts of the Open API Specification:
-* [LINK TO RELEVANT OPEN API SPECS PAGE](https://spec.openapis.org/oas/v3.1.0#data-types)
+### This concerns this parts of the OpenAPI Specification:
+* [EXMAPLE_LINK TO RELEVANT OPEN API SPECS PAGE](https://spec.openapis.org/oas/v3.1.0#data-types)
```
Comment (jtannas) [inline .github/pull_request_template.md]:
```suggestion
* [EXAMPLE_LINK TO RELEVANT OPEN API SPECS PAGE](https://spec.openapis.org/oas/v3.1.0#data-types)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -31,6 +31,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ### Documentation
 
+- Ask for dependency versions in issue tempalte (https://github.com/rswag/rswag/pull/575)
```
Comment (jtannas) [inline CHANGELOG.md]:
```suggestion
- Ask for dependency versions in issue template (https://github.com/rswag/rswag/pull/575)
```

**Review by jtannas — COMMENTED** [review]
2 typos :)
otherwise all good

**Comment by romanblanco** [comment]
@jtannas Thanks!

**Comment by jtannas** [comment]
@romanblanco sorry for not getting back to this sooner. Just ping me when the changelog merge conflict (from the 2.8.0 release) is resolved and I'll merge straight away.

**Comment by romanblanco** [comment]
@jtannas No worries, updated :wink:

**Review by jtannas — APPROVED** [review]

**Pull request merged**